src/length.rs

```diff
@@ -193,6 +194,18 @@ impl<Unit, T: Zero> Zero for Length<T, Unit> {
     }
 }
 
+impl<T, U> Length<T, U>
+where T: Copy + One + Add<Output=T> + Sub<Output=T> + Mul<Output=T> {
+    /// Linearly interpolate between this length and another length.
+    ///
+    /// `t` is expected to be between zero and one.
```

Comment:
I'd rather not. [0..1] is the range that "makes sense" but going beyond is common when when animating things that give a "spring" kind of effect (like the common over-scroll effects on mac). I don't know how to phrase that in a succinct way in the comment, but for most use use cases the interval [0..1] is the one that makes most sense.